modules/assisted-installer-assisted-installer-prerequisites.adoc

```diff
@@ -0,0 +1,62 @@
+// This is included in the following assemblies:
+//
+// installing-on-prem-assisted.adoc
+:_content-type: CONCEPT
+
+[id='assisted-installer-prerequisites_{context}']
+= Assisted Installer prerequisites
+
+Successful installation with the {ai-full} requires ensuring the following prerequisites. The installer will validate these prerequisites before installation.
+
+== Hardware
+
+For control plane nodes or the {sno} node, nodes must have at least:
```

Comment:
actually for non-SNO control plane nodes I see minimal requirement for 4 CPUs and 16GiB RAM
is it intentional to just merge the requirements of SNO and multinode clusters or we should be as correct as possible here?